connexion/operations/abstract.py

```diff
@@ -445,3 +446,10 @@ def json_loads(self, data):
         :type data: bytes
         """
         return self.api.json_loads(data)
+
+    def _get_apikey_parameter(self):
+        security_methods = set().union(*self._security)
+        if 'ApiKeyAuth' in security_methods:
+            api_key_auth = deepcopy(self._security_schemes['ApiKeyAuth'])
```

Comment:
I don't think this key should be hardcoded.
AFAICT 'ApiKeyAuth'  is a user-supplied name for a securityScheme of type `apiKey`.
See this page: https://swagger.io/docs/specification/authentication/api-keys/
Where it calls 'ApiKeyAuth' an "arbitrary name for the security scheme" in the example.

Reply:
Thanks for the quick response. I've updated the code with the apiKey type check.

The implementation does not imply more than one apiKey in query security method for a path. The first found query apiKey will be added to the spec parameters. I also don't see a use case in using more than one query api keys for a path.